f5_os_test/polling_clients.py

```diff
@@ -336,7 +344,92 @@ def delete_all_lbaas_healthmonitors(self):
         return True
 
 
+class HeatClientPollingManager(HeatClient, ClientManagerMixin):
+    '''Utilizes heat client to create/delete heat stacks.'''
+
+    default_stack_config = {
+        'files': {},
+        'disable_rollback': True,
+        'environment': {},
+        'tags': None,
+        'environment_files': None
+    }
+
+    def __init__(self, **kwargs):
+        self.interval = kwargs.pop('interval', 10)
+        self.max_attempts = kwargs.pop('max_attempts', 100)
+
+        super(HeatClientPollingManager, self).__init__(**kwargs)
+
+    def _client(self):
+        return super(HeatClientPollingManager, self)
+
+    def stack_status(self, stack):
+        return stack.stack_status
+
+    def create_stack(self, configuration):
+        configuration.update(self.default_stack_config)
+        stack = self.stacks.create(**configuration)
+        return self.poll(
+            self.stacks.get,
+            stack['stack']['id'],
+            self.stack_status,
+            target_status='CREATE_COMPLETE'
+        )
+
+    def delete_stack(self, stack_id):
+        self.stacks.delete(stack_id)
+        try:
+            self.poll(
+                self.stacks.get,
+                stack_id,
+                self.stack_status,
+                target_status='DELETE_COMPLETE'
+            )
+        except Exception as ex:
```

Comment:
Use 'print(type(ex))' to find out where the expected exception is raised, then import it and catch it specifically, instead of catching a general Exception.